models/stg_klaviyo__event.sql

```diff
@@ -35,10 +35,9 @@ rename as (
         cast(person_id as {{ dbt.type_string() }} ) as person_id,
         type,
         uuid,
-        property_value as numeric_value,
+        {{ remove_string_from_numeric('property_value') }} as numeric_value,
```

Comment:
Once you make the previous comment updates, you will need to change this to the following:
```suggestion
        {{ klaviyo_source.remove_string_from_numeric('property_value') }} as numeric_value,
```

Reply:
Updated!